pkg/plugin/v2/plugin.go

```diff
@@ -117,12 +138,116 @@ func (m *Manager) Start(
 			return fmt.Errorf("error adding client hook for plugin %s: %w", vClusterPlugin.Path, err)
 		}
 
+		// register Interceptors
+		err = m.registerInterceptors(pluginConfig.Interceptors)
+		if err != nil {
+			return fmt.Errorf("error adding interceptor for plugin %s: %w", vClusterPlugin.Path, err)
+		}
+
 		klog.FromContext(ctx).Info("Successfully loaded plugin", "plugin", vClusterPlugin.Path)
 	}
 
 	return nil
 }
 
+// interceptorPortForResource returns the port and handler name for the given group, resource and verb
+func (m *Manager) interceptorPortForResource(group, resource, verb, resourceName string) (ok bool, port int, handlerName string) {
+	groups := m.ResourceInterceptorsPorts
+	if resourcesMap, ok := groups[group]; ok {
+		portHandlerName, ok := portForResource(resourcesMap, resource, verb, resourceName)
+		if ok {
+			return true, portHandlerName.port, portHandlerName.handlerName
+		}
+	}
+	if resourcesMap, ok := groups["*"]; ok {
+		portHandlerName, ok := portForResource(resourcesMap, resource, verb, resourceName)
+		if ok {
+			return true, portHandlerName.port, portHandlerName.handlerName
+		}
+	}
```

Comment:
Could this be shortened to a 
```suggestion
	resourcesMap, ok := groups[group]
	if !ok {
		resourcesMap, ok = groups["*"]
	}
	if !ok {
		return false, 0, ""
	}
	
	portHandlerName, ok := portForResource(resourcesMap, resource, verb, resourceName)
	if !ok {
		return false, 0, ""
	}
	
	return true, portHandlerName.port, portHandlerName.handlerName
```

Reply:
This would also apply to the functions below.